licenses/views.py

```diff
@@ -414,13 +424,61 @@ def view_translation_status(request):
     # branches = [head.name[len("origin/") :] for head in heads]
 
     branches = TranslationBranch.objects.exclude(complete=True)
+
+    legal_code_objects = (
+        LegalCode.objects.valid()
+        .select_related("license")
+        .order_by(
+            "language_code",
+        )
+    )
+    legal_code_langauge_codes = []
+    for lc in legal_code_objects:
+        legal_code_langauge_codes.append(lc.language_code)
```

Comment:
A list comprehension would work here:

```suggestion
    legal_code_langauge_codes = [lc.language_code for lc in language_code_objects]
```